lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
I think you may have misunderstood me. I'm fine with this usage being short_id, but was asking if you can change it in your PR (which I assume is off of this branch). I don't think I want to change all usages of id to short_id in this particular PR.